ISSUE_TEMPLATE.md

```diff
@@ -5,8 +5,10 @@ Having problems? Have you checked these first:
 * If not, please update then uninstall the TinyTDS gem and re-install it.
 * Using Ubuntu? If so, you may have forgotten to install FreeTDS first.
 * Doing work with threads and the raw client? Use the ConnectionPool gem?
+* On Windows? If so, do you need devkit for your ruby install?
 
 If none of these help. Please make sure to report:
 
 * What platform you are on. Windows, Mac, Ubuntu, etc.
-* What version of FreeTDS you are using.
+* What version of TinyTDS are you using?
+* What version of FreeTDS you are using?
```

Comment:
Maybe we have commands to run here after the question. Like:

Run `tsql -C` to see. Optionally show output.

Reply:
These are all good ideas. I'll make these changes and update either tonight or tomorrow. 👍